manifests/localzone.pp

```diff
@@ -26,15 +26,29 @@
 # @param template_name Use a custom template.
 #
 define unbound::localzone (
-  Unbound::Local_zone_type $type,
-  String $zone = $name,
-  $config_file = $unbound::config_file,
-  Array[Unbound::Resource_record_type] $local_data = [],
-  String $template_name = 'unbound/local_zone.erb'
+  Unbound::Local_zone_type             $type,
```

Comment:
I prefer the older style, with just a single whitespace. That makes the diff easier to read and makes git reverts / cherry-picks easier. But there's no consent in our style guide yet.

Reply:
I hear you but the rest of this module uses this style and I prefer consistency.  But consistency across viz would be great so let's try and get something in the style guide and then we can creat a puppet lint check and fix to get everything fixed?